packages/__examples__/.storybook/webpack.config.js

```diff
@@ -23,27 +23,24 @@
  */
 
 const { merge } = require('webpack-merge')
+const baseConfig = require('@instructure/ui-webpack-config')
 
-module.exports = ({ config, mode }) => {
-  const baseConfig = require('@instructure/ui-webpack-config')
-
+module.exports = ({ config }) => {
   // Storybook does not like thread-loader, see
   // https://github.com/storybookjs/storybook/issues/12864
   const rules = baseConfig.module.rules
   for (const rule of rules) {
     if (rule.use) {
       for (let i = 0; i < rule.use.length; i++) {
-        if (rule.use[i].loader === 'thread-loader') {
+        if (
+          rule.use[i].loader &&
+          rule.use[i].loader.includes('thread-loader')
+        ) {
```

Comment:
Since the config now includes the full resolved path to `thread-loader` instead of it being a single string, this filtering logic had to be synchronized.